app/shell/Status/NodeStatus.tsx

```diff
@@ -57,10 +57,16 @@ export default function NodeStatus(): JSX.Element {
             );
             let status = NodeConnectionStatus.Unavailable;
             try {
-                const upToDate = await isNodeUpToDate();
-                status = upToDate
-                    ? NodeConnectionStatus.Ready
-                    : NodeConnectionStatus.CatchingUp;
+                const healthResponse = await healthCheck();
+                if (healthResponse.isHealthy) {
+                    status = NodeConnectionStatus.Ready;
+                } else if (
+                    healthResponse.message?.includes(
+                        'Last finalized block is too far behind'
```

Comment:
Is there no better way to check this? This seems a little fragile against changes to the text. Are there any other types of health responses that would be relevant?

Reply:
There is no other way to check this, the same error code is when we can't connect at all. :cry: